<?xml version="1.0"?>
<robot
    name="test"
    xmlns:xacro="http://www.ros.org/wiki/xacro"
    xmlns:sensor="http://playerstage.sourceforge.net/gazebo/xmlschema/#sensor"
    xmlns:controller="http://playerstage.sourceforge.net/gazebo/xmlschema/#controller"
    xmlns:interface="http://playerstage.sourceforge.net/gazebo/xmlschema/#interface">

    <!--<xacro:include filename="$(find ros_human_gazebo)/humanSubject01/humanSubject01_48dof.urdf"/>-->
    <xacro:include filename="$(find ros_human_gazebo)/humanSubject01/humanSubject01_mod3.urdf"/>

    <link name="world"/>

    <joint name="base_joint" type="floating">
      <parent link="world" />
      <child link="Pelvis" />
      <origin xyz="0 0 0" rpy="0 0 0"/>
    </joint>

    <link name="LeftHandTip"/>
    <joint name="jLeftHandTip" type="fixed">
      <parent link="LeftHand" />
      <child link="LeftHandTip" />
      <origin xyz="0 +0.16718 0" rpy="0 0 0"/>
    </joint>

    <link name="RightHandTip"/>
    <joint name="jRightHandTip" type="fixed">
      <parent link="RightHand" />
      <child link="RightHandTip" />
      <origin xyz="0 -0.16718 0" rpy="0 0 0"/>
    </joint>

</robot>
